### ISSUE TITLE
Binding does not work inside DropDownContent

### ISSUE THREAD

**Issue by zidik** [xceedsoftware/wpftoolkit#1216]
Version 3.2.0

I am having issues binding to a property inside a DropDownContent. The TextBlock stays empty.

```
   <xctk:DropDownButton Content="Click Me" Margin="15" >
        <xctk:DropDownButton.DropDownContent>
            <TextBlock Text="{Binding MyProperty}"/>
        </xctk:DropDownButton.DropDownContent>
    </xctk:DropDownButton>
```

     
Binding just outside the DropDownButton works perfectly - the TextBlock displays the value of MyProperty.
```
    <TextBlock Text="{Binding MyProperty}"/>
```

**Comment by zidik** [comment]
Reverting to V 3.1.0 resolves the problem

**Comment by XceedBoucherS** [comment]
Hi,
This is the same issue as https://github.com/xceedsoftware/wpftoolkit/issues/1214.
Fixed in v3.3.
